### DISCUSSION TITLE
Need Help with Suspended GitHub Account

### DISCUSSION THREAD

**Discussion by sdgrgrs** [community/community#47041] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

I hope this post finds you well. I recently encountered a problem with my GitHub account and I am hoping for some help from the community.

On 2023/02/09, I found out that my GitHub account was suspended. This was surprising because I only created the account on 2023/02/08 and didn't use it for one day. I am a student and not very familiar with GitHub, so I am not sure what might have caused the suspension.

I have tried to reach out to GitHub support but I have yet to receive a response. If anyone in this community has had a similar experience or has any advice on how to get my account reinstated, I would greatly appreciate it.

Thank you for your time and any help you can provide.

**Comment by osawakoki** [comment]
Hello adgrgrs👋

I also got my account suspended.
My ticket which was created 2 weeks ago have not made any action...
I was told that I have to wait more.

So, simply calculating in your case, a reply will come back after 2/25.
I think that is too long, but we have to wait🥺

**Reply by TCIano** [reply]
Hi bro, have you received a reply?

**Reply by osawakoki** [reply]
Not yet...

16 days have passed.
It's devastating if I use GitHub for a business.

**Reply by TCIano** [reply]
> Not yet...
> 
> 16 days have passed. It's devastating if I use GitHub for a business.

I had the same problem as you. If you get a reply, can you let me know here? because I also haven't received a reply for 14 days. Thanks

**Reply by osawakoki** [reply]
Of cource.

I will come back here when I got a reply.

**Reply by osawa-koki** [reply]
Hi @TCIano.

I got my account unlocked!!!
18 days.
probably, you got your account unlocked in a few days.

🐙 osawa koki 🐙

**Reply by TCIano** [reply]
Hi, @osawa-koki congratulations, I have not received an email response, I would like to know the official said why your account will be suspended? thank you!

**Reply by osawa-koki** [reply]
> I would like to know the official said why your account will be suspended?

I don't know, I was just notified that my account was set open.
Currently asking them for the reason.

**Reply by TCIano** [reply]
> I was just notified that my account was set open. 

Is it an email notification?

**Reply by osawa-koki** [reply]
Yes.

A reply will go to an email which you put when you created a ticket.
You should wait for 2 or 3 days bacause I created a ticket 2 days before you.

**Reply by TCIano** [reply]
I hope so, thank you for the reply.

**Reply by osawa-koki** [reply]
I got a reason.

```reply
Thanks for contacting GitHub Support!
Sometimes our abuse detecting systems highlight accounts that need to be manually reviewed.
We've cleared the restrictions from your account, so you have full access to GitHub again.
```

Actually, I am not satisfied with this reply...
They did not clear the reason of suspending my account.
We have to mirror all the repositories in case of getting an account suspended, as long as they have no clear standard for the system...

I attached the request for refund and got refund for 1 month, while they locked my account for 2.5weeks.
If you use Copilot or register Pro plan, you should make a request for the refund.

I hope you have your account unlocked within a few days.
Thanks.

osawa koki

**Reply by TCIano** [reply]
Hi there, @osawa-koki, I haven't received an email response yet and would like to know how many emails you have sent? I would also like to know what you wrote in your emails? Thank you very much!

**Reply by cunzhang2** [reply]
Hi, @osawa-koki ,What date did you have your account suspended? Thanks

**Reply by osawa-koki** [reply]
@TCIano 

> know how many emails you have sent?

I created 3 tickets.

---

> what you wrote in your emails?

1. summary
  - date my account got suspended(2023/01/30)
  - what I did right before(adding gpg key, creating a new repo)
 2. concerning
  - I have two accounts but one is a paid account
  - having some cyber-attack-program repos but they are for security test
 3. some proofs
  - my passport(my account name matches one in the passport -> prooving I am not a bot)
  - the usage of some cyber-attack-program repos

Are you still waiting for a reply?
That's too long...

**Reply by osawa-koki** [reply]
@zone-meen 

> What date did you have your account suspended?

My account got suspended om 2023/01/30.
I created a ticket for that immediately.

And, they unlocked my account on 02/16.

It took 2 weeks and 4 days to take my account back.

**Reply by cunzhang2** [reply]
@osawa-koki

My account @zonemeen was likewise suspended on 2023/01/30.
The first ticket was created on 2023/01/31 and the fourth ticket was created on 2023/02/01.
So far I've created four tickets and haven't received any replies, I'm so sad.

> It took 2 weeks and 4 days to take my account back.

What was the last ticket you created?

**Reply by osawa-koki** [reply]
Hi, @zone-meen.

> The first ticket was created on 2023/01/31

It's too long...
I don't know what is making your reply slow, but you should give them some items which prooves that you never violate TOS.
We can guess your ticket is being proccessed and some steps are with probles because my ticket which was made in the same day was dealt.

---

> What was the last ticket you created?

Do you mean a number of my ticket?
Last one is numbered #2010340.
First one is #1987757.

I hope your account set open soon.

**Reply by cunzhang2** [reply]
Thanks~
